### DISCUSSION TITLE
Existe maneira de interromper a execução?

### DISCUSSION THREAD

**Discussion by CaduAngelo** [dgadelha/Portugol-Webstudio#283] [Dúvidas / Perguntas] [answered]
Gostaria de saber se existe algum comando onde eu possa finalizar a execução do programa sem precisar executar a rotina completa.

**Comment by SrMisterio39** [comment]
Também estou com a mesma dúvida, gostaria muito dessa resposta!

**Comment by dgadelha** [comment, marked as answer]
Olá,

Se o código estiver em execução na função `inicio`, você pode a qualquer momento interrompê-la chamando `retorne`:

```portugol
programa {
  funcao inicio() {
    caracter valor
    escreva("Digite S para Continuar ou N para Interromper: ")
    leia(valor)
    se (valor == 'N') {
      retorne
    }
    escreva("Você continuou!")
  }
}
```

**Reply by SrMisterio39** [reply]
Muito obrigado pela resposta, me ajudou no meu trabalho!

**Comment by CaduAngelo** [comment]
Caramba... eu tentei retorna e retorno hehe Vlw
